documentation/getting-started/getting-started-project-wizard.asciidoc

```diff
@@ -1,79 +1,62 @@
 
-[[getting-started.project-wizard.quick]]
-== Using the Quick Starter Wizard
+Click *Start building* a project for Vaadin 14 LTS or the latest release at the http://vaadin.com/start[Get Started] page.
+It launches the builder at http://start.vaadin.com/[start.vaadin.com].
 
-In the following, we describe how you can create a simple downloadable project with the quick starter wizard.
-The project has a minimalistic UI with an input field and a button, and a service layer stub.
+.Getting Started with Vaadin Flow
+video::C78LjVQhejI[youtube, width=480, height=360]
 
-. Navigate to http://vaadin.com/start[vaadin.com/start].
+The generated applications feature:
 
-. Choose the Vaadin version for the project.
-The latest LTS (Long-Term Service) version is suggested, but you can also choose a newer sversion if one is available.
-+
-.Choosing the Vaadin version.
-image::images/project-wizard-01.png[Project Wizard]
+* A main view with navigation to child views
+* Child views in server-side Java (Flow) or client-side TypeScript (Fusion)
+* Building with Maven
+* Running with Spring Boot embedded server
 
-. Select *Download* (default)
+You can build Fusion (TypeScript) views in Vaadin 15 and later.
 
-. Select a technology stack.
-+
-See the descriptions of the technology stacks in <<getting-started-overview#getting-started.overview.stacks, Technology Stacks>>.
-+
-image::images/project-wizard-02.png[Download]
-
-. Enter a *Maven Group ID*.
-This is usually your corporation ID or a personal ID.
-You are welcome to use one under `org.vaadin`, especially for open-source projects.
-
-. Enter a *Project Name*.
-It will also be used for the project ID in lowercase, with spaces replaced by dashes.
-
-. Click *Download*.
-
-. Unzip the downloaded archive to a file location of your choice.
-+
-TIP: Avoid unzipping to the download folder, as you could unintentionally delete your project when clearing out old downloads.
-
-After downloading the package and uncompressing it, you are now ready to import the project in your IDE, as described in subsequent sections.
-Please see the instructions for your IDE in <<getting-started-eclipse#getting-started.eclipse.importing, Importing a Project in the Eclipse IDE>>, <<getting-started-eclipse#getting-started.eclipse.importing, Importing a Project in IntelliJ>>, or <<getting-started-netbeans#getting-started.netbeans.importing, Importing a Project in NetBeans>>.
-
-[[getting-started.project-wizard.customizing]]
-== Creating a Customized Starter Project
-
-To create customized starter application, click [guibutton]#Customize App# in the project wizard, or navigate directly to the https://start.vaadin.com/[start.vaadin.com] to create a new project.
-The customization wizard allows creating Vaadin 15 projects.
+The builder shows the tools to build the views on the left and the application UI preview on the right.
 
 .Customizing a project
 image::images/project-customizing-01.png[Customizing a Project]
```

Comment:
This screenshot is outdated.

Reply:
I actually updated the screenshot, but didn't change the name. I'll change the title to say builder.